vendor/github.com/intel/govmm/qemu/qemu.go

```diff
@@ -92,6 +92,9 @@ const (
 	//VhostUserBlk represents a block vhostuser device type.
 	VhostUserBlk DeviceDriver = "vhost-user-blk-pci"
 
+	//VhostUserFS represents a virtio-fs vhostuser device type
```

Comment:
Please get these changes into https://github.com/intel/govmm first, and then re-vendor the changes into the runtime (see https://github.com/kata-containers/community/blob/master/VENDORING.md).